sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -31,23 +31,18 @@ public class ProcessMessageEventArgs : EventArgs
         /// Initializes a new instance of the <see cref="ProcessMessageEventArgs"/> class.
         /// </summary>
         ///
-        /// <param name="message"></param>
-        /// <param name="receiver"></param>
-        /// <param name="cancellationToken"></param>
-        internal ProcessMessageEventArgs(ServiceBusReceivedMessage message, ServiceBusReceiver receiver, CancellationToken cancellationToken)
+        /// <param name="message">The message to be processed.</param>
+        /// <param name="receiver">The receiver instance that can be used to perform message settlement.</param>
+        /// <param name = "cancellationToken" > The processor's <see cref="System.Threading.CancellationToken"/> instance which will be cancelled in the event that <see cref="ServiceBusProcessor.StopProcessingAsync"/> is called.
```

Comment:
I'm guessing the extra spaces in the param tag don't affect it, right?

Reply:
I don't think so, but not sure why that happened.